hikari/impl/interaction_server.py

```diff
@@ -133,7 +140,10 @@ def status_code(self) -> int:
 
 # Constant response
 _PONG_RESPONSE: typing.Final[_Response] = _Response(
-    _OK_STATUS, data_binding.dump_json({"type": _PONG_RESPONSE_TYPE}).encode(), content_type=_JSON_TYPE_WITH_CHARSET
+    _OK_STATUS,
+    data_binding.dump_json({"type": _PONG_RESPONSE_TYPE}).encode(),
+    content_type=_JSON_CONTENT_TYPE,
+    charset=_UTF_8_CHARSET,
```

Comment:
We default it to UTF_8_CHARSET, no need to explicitly pass it here

Reply:
Done